Patches/Simple Ogre Race/CE_Ogre_Race.xml

```diff
@@ -0,0 +1,138 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+	<!-- It's all Ogre now -->
+	<Operation Class="PatchOperationFindMod">
+					<mods>
+						<li>Simple Ogre Race</li>
+					</mods>
+		<match Class="PatchOperationSequence">
+			<operations>
+
+			<li Class="PatchOperationAddModExtension">
+				<xpath>/Defs/AlienRace.ThingDef_AlienRace[defName="FP_OgreRace"]</xpath>
+				<value>
+					<li Class="CombatExtended.RacePropertiesExtensionCE">
+						<bodyShape>Humanoid</bodyShape>
+					</li>
+				</value>
+			</li>
+			
+			<!-- Melee Tool & Basestats Defs -->
+			<li Class="PatchOperationReplace">
+				<xpath>/Defs/AlienRace.ThingDef_AlienRace[defName="FP_OgreRace"]/statBases/Mass</xpath>
+					<value>
+      						<Mass>80</Mass>
+						<MeleeDodgeChance>1</MeleeDodgeChance>
+						<MeleeCritChance>1.5</MeleeCritChance>
+						<MeleeParryChance>1</MeleeParryChance>
+          						<Suppressability>0.75</Suppressability>
+						<CarryWeight>55</CarryWeight>
+			                			<CarryBulk>30</CarryBulk>
+					</value>
+			</li>
+
+			<li Class="PatchOperationReplace">
+				<xpath>/Defs/AlienRace.ThingDef_AlienRace[defName="FP_OgreRace"]/statBases/MoveSpeed</xpath>
+				<value>
+					<MoveSpeed>4.0</MoveSpeed>
+				</value>
+			</li>
+
+			<li Class="PatchOperationReplace">
+				<xpath>/Defs/AlienRace.ThingDef_AlienRace[defName="FP_OgreRace"]/statBases/ArmorRating_Sharp</xpath>
+				<value>
+					<ArmorRating_Sharp>0.2</ArmorRating_Sharp>
+				</value>
+			</li>
+
+			<li Class="PatchOperationReplace">
+				<xpath>/Defs/AlienRace.ThingDef_AlienRace[defName="FP_OgreRace"]/statBases/ArmorRating_Blunt</xpath>
+				<value>
+					<ArmorRating_Blunt>2</ArmorRating_Blunt>

```

Comment:
This amount of armor does make them largely immune to damage from both unarmed humans and a number of animals. Is this intended?

Reply:
Yes, this is more of an issue of unarmed human being too weak, if unarmed unarmour pen also scales with melee skill, that'd be great.